permissions/component-pipeline-model-json-shaded.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-model-json-shaded"
+paths:
+- "org/jenkinsci/plugins/pipeline-model-json-shaded"
```

Comment:
`or/jenkins-ci/lib` seems like a straightforward choice unless I'm mistaking what this is for.